json_spec/input/redfish_response_messages.json

```diff
@@ -0,0 +1,4 @@
+{
+    "UpdateProgress_Message":"UpdateInProgress",
+    "LargeFWImageUpdate": "PayloadTooLarge"
```

Comment:
@v-ajitpanda Details are missing in README.md for all new config/files. Please update README.md for all new changes.Thanks.

Reply:
Removed debug statements

Reply:
@v-ajitpanda Please update the README as well. Thanks.

Reply:
Looks good, test cases passing at our end.